applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -361,6 +361,11 @@ def _add_temperature_variables(self):
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.DT_TEMPERATURE)
         # Add variables for the heat conditions
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.NORMAL_HEAT_FLUX)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.AIR_TEMPERATURE)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.SOLAR_RADIATION)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.AIR_HUMIDITY)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.PRECIPITATION)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.WIND_SPEED)        
```

Comment:
Why the difference in indentation?

Reply:
For me it looks okay, maybe it's a github issue?
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/d74dbf9c-affb-4537-ba3d-5c780fa9c1df)